utils/main.go

```diff
@@ -187,6 +187,10 @@ func SaferWriteFile(content []byte, outputFile string) error {
 		return fmt.Errorf("unable to create temporary file under %q: %w", dir, err)
 	}
 
+	if err := os.Chmod(tmp.Name(), 0644); err != nil {
```

Comment:
can we make it even more generic and add a `mode` argument to `SaferWriteFile()`?

Reply:
done